x/sequencer/keeper/msg_server_kick_proposer_test.go

```diff
@@ -1,13 +1,14 @@
 package keeper_test
 
 import (
+	rollapptypes "github.com/dymensionxyz/dymension/v3/x/rollapp/types"
 	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
 	"github.com/dymensionxyz/gerr-cosmos/gerrc"
 	"github.com/dymensionxyz/sdk-utils/utils/utest"
 )
 
 func (s *SequencerTestSuite) TestKickProposerBasicFlow() {
-	s.App.RollappKeeper.SetHooks(nil)
+	s.App.RollappKeeper.SetHooks(rollapptypes.NewMultiRollappHooks(s.k().RollappHooks()))
```

Comment:
I've made sure that the new panic is triggered, when the new bug fix is not included. To do that, hooks are needed.